specificatie/openapi.yaml

```diff
@@ -1200,21 +1079,19 @@ components:
     Beslag_links:
       type: "object"
       properties:
-        beslaglegger:
-         $ref: "https://raw.githubusercontent.com/VNG-Realisatie/Haal-Centraal-common/v1.0.0/api-specificatie/common.yaml#/components/schemas/HalLink"
-        stukken:
+        beslagleggers:
           type: "array"
           items:
             $ref: "https://raw.githubusercontent.com/VNG-Realisatie/Haal-Centraal-common/v1.0.0/api-specificatie/common.yaml#/components/schemas/HalLink"
-        isGebaseerdOpStukdeel:
+        stukken:
           type: "array"
           items:
             $ref: "https://raw.githubusercontent.com/VNG-Realisatie/Haal-Centraal-common/v1.0.0/api-specificatie/common.yaml#/components/schemas/HalLink"
-        isVermeldInStukdelen:
+        isGebaseerdOpStukdeel:
```

Comment:
moet dit dan niet zijn "isGebaseerdOpStukdelen"? Het is immers een array

Reply:
Moet geen array zijn. Ook gewijzigd bij Hypotheek_links